packages/api/src/utils/createTask.ts

```diff
@@ -1073,4 +1073,23 @@ export const enqueueExpireFoldersJob = async () => {
   )
 }
 
+export const queueExportJob = async (userId: string) => {
+  const queue = await getQueue()
+  if (!queue) {
+    return undefined
+  }
+
+  return queue.add(
+    EXPORT_JOB_NAME,
+    { userId },
+    {
+      jobId: `${EXPORT_JOB_NAME}_${userId}_${JOB_VERSION}`,
+      removeOnComplete: true,
+      removeOnFail: true,
```

Comment:
The export jobs are deduplicated but we probably also want to limit number of exports per user per day by checking the number of zip files in cloud storage

Reply:
Yeah agreed, or even create an entry in postgres, as annoying as that can be

Reply:
Yeah, makes sense to me